tsinfer/formats.py

```diff
@@ -435,6 +435,13 @@ def _check_write_modes(self):
         if self._mode not in (self.EDIT_MODE, self.BUILD_MODE):
             raise ValueError("Invalid operation: must be in edit or build mode")
 
+    def _check_finalised(self):
+        if not self.finalised:
+            error_msg = "The {} file".format(self.format_name)
```

Comment:
I'm pretty sure we should change `"The {} file"` --> `"The {} file "`, with a space after "file"

Reply:
Good point